src/script/pages/app-home.ts

```diff
@@ -28,7 +28,7 @@ export class AppHome extends LitElement {
         font-size: 16px;
         line-height: 24px;
         letter-spacing: -0.015em;
-        color: #A6A4A4;
+        color: #000000 50%;
```

Comment:
good candidate for a css variabl

Reply:
Agreed, will add.

Reply:
Updated below in a new commit, going to resolve this. Thanks Leo (: